CONTRIBUTE.md

```diff
@@ -0,0 +1,333 @@
+# JavaScript Interactive Examples Contributing Guide
```

Comment:
I think "Style Guide" would be better than "Contributing Guide". There's a lot else to contributing than the style of the examples, which is all that this guide covers.

Reply:
:+1: